dev-python/rtslib-fb/rtslib-fb-2.1.73.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit distutils-r1 systemd
+
+DESCRIPTION="A Python object API for managing the Linux LIO kernel target"
+HOMEPAGE="https://github.com/open-iscsi/rtslib-fb"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~x86 ~amd64-linux ~x86-linux"
+IUSE=""
```

Comment:
You can drop empty IUSE (see [PMS](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-620007.3)).